src/components/Reviews/Reviews.jsx

```diff
@@ -133,46 +145,62 @@ class Reviews extends Component {
     return (
       <ReviewsWrapper>
         <GlobalStyle />
-        <ReviewsOverview
-          reviews={reviews}
-          totalReviewsCount={totalReviewsCount}
-          reviewStarPercentages={reviewStarPercentages}
-          reviewAverageScore={reviewAverageScore}
-          filterCondition={filterCondition}
-          showLessScroll={this.showLessScroll}
-          filterReviewsByStarRating={this.filterReviewsByStarRating}
-          scrollToReviewsBody={this.scrollToReviewsBody}
-        />
-        {customerImages && (
-          <ReviewsCustomerPhotos
-            customerImages={customerImages}
-          />
-        )}
-        <ReviewsSearch
-          filterReviewsByText={this.filterReviewsByText}
-        />
-        <ReviewsSort
-          reviewDisplayCount={reviewDisplayCount}
-          filteredReviews={filteredReviews}
-          filterCondition={filterCondition}
-          filterReviewsByStarRating={this.filterReviewsByStarRating}
-          sortReviews={this.sortReviews}
-        />
-        <ReviewsDisplay
-          reviewDisplayCount={reviewDisplayCount}
-          filteredReviews={filteredReviews}
-          filterCondition={filterCondition}
-          reviewScroll={this.reviewScroll}
-          increaseReviewDisplayCount={this.increaseReviewDisplayCount}
-          resetReviewDisplayCount={this.resetReviewDisplayCount}
-        />
-        <ReviewsShowMoreOrLess
-          reviewDisplayCount={reviewDisplayCount}
-          filteredReviewsLength={filteredReviews.length}
-          increaseReviewDisplayCount={this.increaseReviewDisplayCount}
-          resetReviewDisplayCount={this.resetReviewDisplayCount}
-          scrollToReviewsOverview={this.scrollToReviewsOverview}
-        />
+        <section>
+          <ReviewsHeader showLessScroll={this.showLessScroll} />
+          {!reviewsLoaded ? <MainLoadingPlaceholder /> : (
+            <ReviewsOverview
+              reviews={reviews}
+              totalReviewsCount={totalReviewsCount}
+              reviewStarPercentages={reviewStarPercentages}
+              reviewAverageScore={reviewAverageScore}
+              filterCondition={filterCondition}
+              filterReviewsByStarRating={this.filterReviewsByStarRating}
+              scrollToReviewsBody={this.scrollToReviewsBody}
+            />
+          )}
+          {!reviewsLoaded ? <BarsLoadingPlaceholder /> : (
+            <section>
+              {customerImages && (
+                <ReviewsCustomerPhotos
+                  customerImages={customerImages}
+                />
+              )}
+            </section>
+          )}
+          {!reviewsLoaded ? null : (
```

Comment:
It may seem like a silly question, but would it be possible to refactor lines 170-202 to be only one ternary operator? That might clean it up a bit.

Reply:
Nice one!